tests/test_benchmark/test_benchmark.py

```diff
@@ -11,24 +12,56 @@
 
 import mteb
 from mteb.benchmarks import Benchmark
-
-from .mock_models import MockNumpyEncoder, MockTorchbf16Encoder, MockTorchEncoder
+from mteb.cli import create_meta
+
+from .mock_models import (
+    MockBGEWrapper,
+    MockE5Wrapper,
+    MockMxbaiWrapper,
+    MockNumpyEncoder,
+    MockTorchbf16Encoder,
+    MockTorchEncoder,
+)
 from .task_grid import MOCK_TASK_TEST_GRID
 
 logging.basicConfig(level=logging.INFO)
 
 
-@pytest.mark.parametrize("tasks", [MOCK_TASK_TEST_GRID[:2]])
+@pytest.mark.parametrize("tasks", [MOCK_TASK_TEST_GRID])
 @pytest.mark.parametrize("model", [MockNumpyEncoder()])
-def test_mulitple_mteb_tasks(tasks: list[mteb.AbsTask], model: mteb.Encoder):
+def test_mulitple_mteb_tasks(
+    tasks: list[mteb.AbsTask], model: mteb.Encoder, monkeypatch
+):
     """Test that multiple tasks can be run"""
     eval = mteb.MTEB(tasks=tasks)
-    eval.run(model, output_folder="tests/results", overwrite_results=True)
+    output_folder = "tests/results"
+    eval.run(model, output_folder=output_folder, overwrite_results=True)
+
+    tasks_dict = {task.metadata.name: task for task in tasks}
+
+    # make sure that all tasks results can be exported
+    args = Namespace(
+        results_folder=output_folder + "/no_model_name_available/no_revision_available",
+        output_path=output_folder + "/model_card.md",
+        overwrite=True,
+    )
+    monkeypatch.setattr(
+        mteb, "get_task", lambda task_name, **kwargs: tasks_dict[task_name]
+    )
+    create_meta(args)
```

Comment:
I'm not sure that this is a good place for test exporting results, but I don't know where to put it.

Reply:
It seems like you have replaced an existing test, and it now tests something else. I would instead create a new test. I also don't believe that there is a need to monkey patch -- generally, I find that it hints that the code should be rewritten.

What I would do is create a function like:

```
create_meta_cli(args):
  # handle args, get tasks etc. 
  yml_dict = create_meta(task_results) 
  # save to disk
```

We can then test `create_meta` independently without monkeypatching.

Reply:
This is not breaking as the create_meta cli will remain the same, while the create_meta becomes a testable function that the user can also call independently.

Reply:
I'll move it to different PR

Reply:
I needed to monkeypatch it, because create_meta checks if task is existing mteb task, but mock tasks not in main part of library so it gives errors.

Reply:
Yeah, I can definitely see why you added it, but I think the rewrite is the better option. I can also see why you would do the monkey patch for the PR as well (it is generally less obtrusive than changing code).